Resources/Prototypes/Entities/Objects/Tools/welders.yml

```diff
@@ -84,6 +84,8 @@
         - ReagentId: WeldingFuel
           Quantity: 250
         maxVol: 250
+  - type: StaticPrice
+    price: 1755
```

Comment:
these are too high, i dont want these purchasable anyways these should be rewards/loot/science craftable, and should only be just slightly more valuable than their basic counterparts.